logstash/README.md

```diff
@@ -0,0 +1,61 @@
+# Loki Logstash Output Plugin
+
+Logstash plugin to send logstash aggregated logs to Loki.
+
+## Install dependencies
+
+First you need to setup JRuby environment to build this plugin. Refer https://github.com/rbenv/rbenv for setting up your rbenv environment.
+
+After setting up `rbenv`. Install JRuby
+
+```bash
+rbenv install jruby-9.2.10.0
+rbenv local jruby-9.2.10.0
+```
+
+Check that the environment is configured
+
+```bash
+ruby --version
+jruby 9.2.10
+```
+
+You should use make sure you are running jruby and not ruby.
+
+Then install bundler
+`gem install bundler:2.1.4`
```

Comment:
This should be `<2`. I think this was the reason `bundle exec rspec` failed. This is what everyone are using
https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/.ci/Dockerfile#L18

https://github.com/logstash-plugins/logstash-output-kafka/blob/master/.travis.yml#L21

Reply:
ok !